tests/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Will show up as Issue #7 PR (once this one is merged) - https://github.com/GoogleCloudPlatform/jetty-runtime/commit/a8b9aa7f1dabbed0d86ae9580416f500e89f9823